docs/gallery/using_nisq_circuits.ipynb

```diff
@@ -0,0 +1,530 @@
+    "    if block_graph_for_computation:\n",
+    "        compiled_graph = compile_block_graph(block_graph_for_computation)\n",
+    "        stim_circuit = compiled_graph.generate_stim_circuit(\n",
+    "            k=1, noise_model=NoiseModel.uniform_depolarizing(p=0.001)\n",
+    "        )\n",
+    "        print(\"\\nFirst 10 lines of Stim circuit (Basis X):\\n\")\n",
+    "        print(stim_circuit[:10])\n",
+    "    \n",
+    "    block_graph_for_computation = graphs_for_given_basis(pre_filled_block_graphs, Basis.Z)\n",
+    "    if block_graph_for_computation:\n",
+    "        compiled_graph = compile_block_graph(block_graph_for_computation)\n",
+    "        stim_circuit = compiled_graph.generate_stim_circuit(\n",
+    "            k=1, noise_model=NoiseModel.uniform_depolarizing(p=0.001)\n",
+    "        )\n",
+    "        print(\"\\nFirst 10 lines of Stim circuit (Basis Z):\\n\")\n",
+    "        print(stim_circuit[:10])\n",
+    "\n",
+    "# Call Stim circuit generation for all relevant bases\n",
+    "if filled_block_graphs:\n",
+    "    get_stim_circuit(filled_block_graphs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "497faf0f",
+   "metadata": {},
+   "source": [
+    "And let's now run a simulation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "503d3333",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Function to generate simulation graphs\n",
+    "def generate_graphs(pre_filled_block_graphs, support_observable_basis: Basis, SAVE_DIR) -> None:\n",
+    "\n",
+    "    # Get block_graph from set of pre_filled block_graphs\n",
+    "    block_graph_for_simulation = graphs_for_given_basis(pre_filled_block_graphs, support_observable_basis)\n",
+    "    \n",
+    "    # Start sinter\n",
+    "    if block_graph_for_simulation:\n",
+    "\n",
+    "        # ZX-graph to overlay over output\n",
+    "        zx_graph_for_simulation = block_graph_for_simulation.to_zx_graph()\n",
+    "\n",
+    "        # Correlation surfaces for simulation\n",
+    "        correlation_surfaces_for_simulation = block_graph_for_simulation.find_correlation_surfaces()\n",
+    "        \n",
+    "        # Call Sinter\n",
+    "        stats = start_simulation_using_sinter(\n",
+    "            block_graph_for_simulation,\n",
+    "            range(1, 4),\n",
+    "            list(np.logspace(-4, -1, 10)),\n",
+    "            NoiseModel.uniform_depolarizing,\n",
+    "            manhattan_radius=2,\n",
+    "            observables=correlation_surfaces_for_simulation,\n",
+    "            max_shots=1_000_000,\n",
+    "            max_errors=5_000,\n",
+    "            decoders=[\"pymatching\"],\n",
+    "            print_progress=False,\n",
+    "            save_resume_filepath=Path(\n",
+    "                f\"../_examples_database/steane_stats_{support_observable_basis.value}.csv\"\n",
```

Comment:
what are your thoughts on renaming this? There already exists a database with this name due to `docs/gallery/steane_encoding.ipynb`